.github/build-mandrel-images.sh

```diff
@@ -17,5 +17,12 @@ do
         ${BUILD_ENGINE} --tag="${PREFIX_NAME}:${version}"
 done
 
+# Get all the final versions, reverse sort them, and keep only the first for each major.minor (e.g. 20.1)
+LATEST_FINAL_VERSIONS=$(sed 's/ /\n/g' <<< ${VERSIONS[@]} | grep Final | sort -r | sort -k1,1 -k2,2 -k5,5 -t'.' --unique)
```

Comment:
Just "wow" ... :-D

Reply:
Let's hope the next one who will want to alter/understand it will have the same reaction :laughing: